src/main/java/com/coveros/selenified/element/WaitFor.java

```diff
@@ -57,8 +57,8 @@
     private static final String DISPLAYED = " to be displayed";
     private static final String ENABLED = " to be enabled";
 
+    private static final long defaultPollingInterval = 50;
```

Comment:
![CRITICAL](https://sonarsource.github.io/sonar-github/severity-critical.png 'Severity: CRITICAL') Rename this constant name to match the regular expression '^[A-Z][A-Z0-9]*(_[A-Z0-9]+)*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](http://localhost:9000/sonar/coding_rules#rule_key=squid%3AS00115)